test/acceptance/auto_auth.py

```diff
@@ -82,11 +82,24 @@ def get_user_id(self):
 
     def get_username(self):
         """
-        Finds and returns the username
+        Finds and returns the username of the current user.
+        """
+        username_and_email = self._get_username_and_email()
+        return username_and_email.split(' ')[0]
+
+    def get_email(self):
+        """
+        Finds and returns the email address of the current user.
+        """
+        username_and_email = self._get_username_and_email()
+        return username_and_email.split(' ')[1]
+
+    def _get_username_and_email(self):
+        """
+        Finds and returns a string containing username and email address of the current user.
         """
```

Comment:
I think it makes sense to return the username and email as a tuple instead of as a string.